frontend/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I've already created a .gitignore in the root repository. 
Either move the old one to the backend folder or merge the old one with this one

Do it any time u want, i think this can be merged as it is for now as well